src/components/inputs/file/file.component.tsx

```diff
@@ -87,6 +111,13 @@ const File: React.FC<FormFieldInputProps> = ({ field, value, setFieldValue }) =>
             {t('cameraCapture', 'Camera capture')}
           </Button>
         </div>
+        {value && (
```

Comment:
This shouldn't be available in "view mode";

```tsx
{value &&  !isViewMode(sessionMode) && (

```

(Import `isViewMode(mode)` from `../utils/common-utils`)